examples/k8s-health-check/proxy_with_http_health_check.yaml

```diff
@@ -0,0 +1,116 @@
+# You must configure probes in your deployment to use health checks in Kubernetes.
```

Comment:
It would be nice to add a README with an explanation of how to use these health checks.